convert.py

```diff
@@ -22,37 +27,53 @@ def usage():
 Usage:
     {} <account> <infile>
 """.format(__file__)
-    print(s)
+    logging.critical(s)
     sys.exit(1)
 
 
 def check_env():
+    """
+    Checks to ensure important files are present in the correct place.
+    """
+    logging.debug("Commenced function 'check_env'")
     files = [CONFIG_FILE, LEDGER_FILE]
     for f in files:
+        logging.debug("Checking file {}".format(f))
         if (not os.path.exists(f)):
-            print('Cannot find expected file {}'.format(f))
+            logging.critical('Cannot find expected file {}'.format(f))
```

Comment:
Why not using f-strings? I think we can safely force people to use Python 3.6 and above.

Reply:
Concur. Will implement